.github/workflows/ci.yml

```diff
@@ -0,0 +1,109 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+env:
+  MPI_DIR: /usr
+  DAGMC_ROOT: $HOME/DAGMC
+  HDF5_ROOT: /usr
+  OMP_NUM_THREADS: 2
+  OPENMC_CROSS_SECTIONS: $HOME/nndc_hdf5/cross_sections.xml
+  OPENMC_ENDF_DATA: $HOME/endf-b-vii.1
+  COVERALLS_PARALLEL: true
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+
+jobs:
+  main:
+    runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        python-version: [3.8]
+        mpi: [n, y]
+        omp: [n, y]
+        dagmc: [n]
+        event: [n]
+        vectfit: [n]
+
+        include:
+          - python-version: 3.6
+            omp: n
+            mpi: n
+          - python-version: 3.7
+            omp: n
+            mpi: n
+          - dagmc: y
+            python-version: 3.8
+            mpi: y
+            omp: y
+          - event: y
+            python-version: 3.8
+            omp: y
+            mpi: n
+          - vectfit: y
+            python-version: 3.8
+            omp: n
+            mpi: y
+
+    steps:
+      -
+        uses: actions/checkout@v2
+
+      - 
+        name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      -
+        name: Environment Variables
+        run: |
+          echo "OPENMC_CROSS_SECTIONS=$HOME/nndc_hdf5/cross_sections.xml" >> $GITHUB_ENV
+          echo "OPENMC_ENDF_DATA=$HOME/endf-b-vii.1" >> $GITHUB_ENV
```

Comment:
Shouldn't these be picked up from the `env:` block above?

Reply:
Ah, yeah they should. Removing...